server/bin/pbench-unpack-tarballs.py

```diff
@@ -39,18 +38,23 @@ def main(cfg_name):
     # DB access.
     init_db(config, logger)
 
-    BUCKET = str(sys.argv[1])
+    try:
+        BUCKET = str(sys.argv[1])
+    except Exception as exc:
+        logger.error("Bucket not Defined: ", exc)
+        return 1
```

Comment:
If we're going to use a `try` block, then we can't "just continue" -- we have to set `BUCKET` to _something_...otherwise, we'll get a `NameError` when we reference it later....

Reply:
Details, shmetails. `BUCKET = None` would work fine, and subsequent code will use the default lower and upper bounds.